src/systems/ec/implementations/react/components/site-header-core/src/SiteHeaderCore.jsx

```diff
@@ -1,13 +1,17 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 import Icon from '@ecl/ec-react-component-icon';
+import Link from '@ecl/ec-react-component-link';
 import SearchForm from '@ecl/ec-react-component-search-form';
 import { LanguageListOverlay } from '@ecl/ec-react-component-language-list';
 
 const SiteHeaderCore = ({
   logo,
+  logged,
+  loginToggle,
+  loginBox,
```

Comment:
An additional `onLogout` handler executed on clicking on the log out would be appreciated. This way the consumer of the component can pass down a custom handler from parent component which will kill session (side effect) and still keep this component pure.

Reply:
As far as I know, login and logout will be handled by EU login, so it will probably be a link with some parameters.
I agree that adding handlers could be a good improvement (not only for this component), but I'd rather create a nice to have ticket for this